src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -74,18 +52,18 @@ export default class VivViewerLayer extends CompositeLayer {
       // https://github.com/uber/deck.gl/blob/3f67ea6dfd09a4d74122f93903cb6b819dd88d52/modules/geo-layers/src/tile-layer/tile-layer.js#L50
       updateTriggers: {
         getTileData: [loader]
-      }
+      },
+      onTileError: onTileError || loader.onTileError
```

Comment:
We let the end user to override onTileError if by providing onTileError as a prop to VivViewerLayer, which is what deck.gl TileLayer does.